hawkular-inventory-rest-api/src/main/java/org/hawkular/inventory/rest/json/RelationshipEmbeddedJacksonSerializer.java

```diff
@@ -104,7 +104,7 @@ private void serializeEntity(CanonicalPath entity, JsonGenerator jsonGenerator,
         jsonGenerator.writeStartObject();
 
         jsonGenerator.writeFieldName(FIELD_ID);
-        jsonGenerator.writeString(EntityIdUtils.getStableId(entity));
+        jsonGenerator.writeString(Inventory.getStableId(entity));
```

Comment:
This actually should go away, because an entity cannot be found using the stable id - stable id is merely an identifier used to find the security details of the entity, but NOT the entity itself in inventory. The stable id is not stored in inventory and the full canonical path is not recoverable from the stable id.